cli/app.go

```diff
@@ -1051,11 +1091,22 @@ var app = &cli.App{
 											Name:  trainFlagModelVersion,
 											Usage: "version of ML model. defaults to current timestamp if unspecified.",
 										},
+										&cli.StringFlag{
+											Name:     mlTrainingFlagURL,
+											Usage:    "url of Github repository associated with the training scripts",
+											Required: false,
+										},
```

Comment:
(flyby) added this flag, which was being searched for in the `withUpload` method but wasn't settable.
@etai-shuchatowitz @tahiyasalam I copied the flag details from other commands, if you have any issues with the shape of the flag here, please let me know!

Reply:
great, thank you!